pennylane/ops/op_math/condition.py

```diff
@@ -668,7 +668,7 @@ def _validate_abstract_values(
             )
 
 
-def _get_mcm_predicates(conditions: tuple[MeasurementValue]) -> list[MeasurementValue]:
+def get_mcm_predicates(conditions: tuple[MeasurementValue]) -> list[MeasurementValue]:

```

Comment:
If we're going to make this public, should we flesh out the docstring a bit more?